src/remote/types.ts

```diff
@@ -0,0 +1,56 @@
+import type { QueueSong } from '/@/renderer/api/types';
+import type { SongUpdate } from '/@/renderer/types';
+
+export interface SongUpdateSocket extends Omit<SongUpdate, 'song'> {
+    song?: QueueSong | null;
+}
+
+export interface ServerError {
+    data: string;
+    event: 'error';
+}
+
+export interface ServerFavorite {
+    data: { favorite: boolean; id: string };
+    event: 'favorite';
+}
+
+export interface ServerProxy {
+    data: string;
+    event: 'proxy';
+}
+
+export interface ServerRating {
+    data: { id: string; rating: number };
+    event: 'rating';
+}
+
+export interface ServerSong {
+    data: SongUpdateSocket;
+    event: 'song';
+}
+
+export type ServerEvent = ServerError | ServerFavorite | ServerRating | ServerSong | ServerProxy;
+
+export interface ClientSimpleEvent {
+    event: 'next' | 'pause' | 'play' | 'previous' | 'proxy' | 'repeat' | 'shuffle';
+}
```

Comment:
Should the simple event also include "stop" or is not possible to stop the player and instead only pause?

Reply:
I believe the player is designed such that a 'stop' stops the MPV player (but in the logs it's reported as a pause, and the MPRIS connector maps stop to pause as well).

Reply:
Got it, that makes sense